src/apb/engine.py

```diff
@@ -632,3 +747,39 @@ def cmdrun_bootstrap(**kwargs):
         exit(1)
 
     print("Successfully bootstrapped Ansible Service Broker")
+
+
+def cmdrun_test(**kwargs):
+    project = kwargs['base_path']
+    spec = get_spec(project)
+
+    update_dockerfile(project)
+
+    if not kwargs['tag']:
+        tag = spec['image']
+    else:
+        tag = kwargs['tag']
+    print("Building APB using tag: [%s]" % tag)
+
+    client = docker.DockerClient(base_url='unix://var/run/docker.sock', version='auto')
+    client.images.build(path=project, tag=tag)
+
+    print("Successfully built APB image: %s" % tag)
+
+    # run image that was just created.
+    create_image_pod(tag)
```

Comment:
I would add a check here if the create_image_pod resulted in an exception and exit

Reply:
I am catching all the expectations for create_image_pod, and I think it will still need to run some clean up potentially. 
```python
def create_image_pod(image_name):
      try:                                                                         
          .....                                                
      except Exception as e:                                                       
          pod_name = None                                                          
          print("failed - %s" % e)
```

Reply:
Ah I see how it runs through the cleanup process if it fails so what you have makes sense. LGTM.